docker-compose.yml

```
@@ -0,0 +1,62 @@
+version: '3.8'
+
+services:
+  # Сервис PostgreSQL
+  db:
+    image: postgres:13
+    env_file: .env
+    environment:
+      POSTGRES_DB: ${DB_NAME}
+      POSTGRES_USER: ${DB_USER}
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    ports:
+      - "5432:5432"
+
+  # Сервис Redis
+  redis:
+    image: redis:7
```

Также лучше использовать alpine сборки
```suggestion
    image: redis:7-alpine
```